src/main/java/ch/njol/skript/util/Utils.java

```diff
@@ -714,5 +715,37 @@ public static long parseLong(final String s) {
 		}
 		return lastIndex;
 	}
+
```

Comment:
These things should be in another class, e.g. EntityUtil

Reply:
In that case, the map can probably be loaded in a static block, and the field can be renamed to upper snake case (actually according to Skript conventions, it can in its current form also be renamed to it)